pkg/providers/tinkerbell/comparator_test.go

```diff
@@ -0,0 +1,106 @@
+package tinkerbell_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/gomega"
+	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
+	controlplanev1 "sigs.k8s.io/cluster-api/controlplane/kubeadm/api/v1beta1"
+
+	"github.com/aws/eks-anywhere/pkg/providers/tinkerbell"
+	"github.com/aws/eks-anywhere/pkg/utils/ptr"
+)
+
+func TestKubeadmControlPlaneReplicas(t *testing.T) {
+	g := NewWithT(t)
+
+	KCPComparator := &tinkerbell.KubeadmControlPlane{comparatorKubeadmControlPlane()}
+
+	g.Expect(KCPComparator.Replicas()).To(Equal(1))
+}
+
+func TestKubeadmControlPlaneVersion(t *testing.T) {
+	g := NewWithT(t)
+
+	KCPComparator := &tinkerbell.KubeadmControlPlane{comparatorKubeadmControlPlane()}
+
+	g.Expect(KCPComparator.Version()).To(Equal("1.22"))
```

Comment:
nit: we can use kube versions from [here](https://github.com/aws/eks-anywhere/blob/main/pkg/api/v1alpha1/cluster_types.go#L651-L661) in the places we are hardcoding the version in this test file.